app/controllers/album.js

```diff
@@ -18,12 +20,37 @@ exports.getPhotos = (req, res, next) => {
     albumId: req.params.id
   };
   logger.info(`${message.PREVIOUS_MESSAGE} to list of images of an album by the id: ${params.albumId}`);
-  return albumsService
-    .getPhotosBy(params)
+  return getPhotosBy(params)
     .then(response => {
       const albumImgUrl = response.map(({ url }) => url);
       logger.info(message.MESSAGE_OK);
       return res.status(200).send(albumImgUrl);
     })
     .catch(next);
 };
+
+exports.buyAlbums = (req, res, next) => {
+  const qs = {
```

Comment:
why do you do this?
you can pass req.params to findAlbumBy

Reply:
I did it so the method is general, plus I'm also using it below in getAlbums.

Reply:
Its ok but not neccesary. You are just copying req.params object into a new one called qs